libs/designsystem/src/lib/helpers/keyboard-handler.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
```

Comment:
Should this file perhaps be moved to the dropdown component folder? 
It seems to be very specific to the dropdown component

Reply:
I've moved it to the dropdown folder instead 😄